src/components/Event/Event.st.css

```diff
@@ -0,0 +1,143 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, applyOpacity, fallback, font;
+}
+
+:import {
+    -st-from: "../Text/Text.st.css";
+    -st-default: TPAText;
+}
+
+/*Defaults*/
+:vars {
+    DefaultFontStyle: "{theme: 'Body-M', size: '14px', lineHeight: '1em'}";
+    DefaultTimeColor: color-4;
+    DefaultTitleColor: color-5;
+    DefaultAllDayBackground: color-8;
+    DefaultAllDayOpacity: 0.2; 
+    DefaultTopPadding: 4px;
+    DefaultRightPadding: 4px;
+    DefaultBottomPadding: 4px;
+    DefaultLeftPadding: 12px;
+    DefaultRTLTopPadding: 4px;
+    DefaultRTLRightPadding: 12px;
+    DefaultRTLBottomPadding: 4px;
+    DefaultRTLLeftPadding: 4px;
+    BeforeColor: color(color-4);
+    BeforeOpacity: 0.06;
+}
+
+/*Overrides*/
+:vars {
+    FontStyle: --overridable;
+    TimeColor: --overridable;
+    TitleColor: --overridable;
+    AllDayBackground:  --overridable;
+    AllDayOpacity: --overridable;
+    TopPadding: --overridable;
+    RightPadding: --overridable;
+    BottomPadding: --overridable;
+    LeftPadding: --overridable;
+    RTLTopPadding: --overridable;
+    RTLRightPadding: --overridable;
+    RTLBottomPadding: --overridable;
+    RTLLeftPadding: --overridable;
+}
+
+/* Fallbacks */
+:vars {
+    CurrentFontStyle: fallback(value(FontStyle), font(value(DefaultFontStyle)));
+    CurrentTimeColor: color(fallback(value(TimeColor), value(DefaultTimeColor)));
+    CurrentTitleColor: color(fallback(value(TitleColor), value(DefaultTitleColor)));
+    CurrentAllDayBackground: color(fallback(value(AllDayBackground), value(DefaultAllDayBackground)));
+    CurrentAllDayOpacity: fallback(value(AllDayOpacity), value(DefaultAllDayOpacity));
+    CurrentTopPadding: fallback(value(TopPadding),value(DefaultTopPadding));
+    CurrentRightPadding: fallback(value(RightPadding),value(DefaultRightPadding));
+    CurrentLeftPadding: fallback(value(LeftPadding),value(DefaultLeftPadding));
+    CurrentBottomPadding: fallback(value(BottomPadding),value(DefaultBottomPadding));
+    CurrentRTLTopPadding: fallback(value(RTLTopPadding),value(DefaultRTLTopPadding));
+    CurrentRTLRightPadding: fallback(value(RTLRightPadding),value(DefaultRTLRightPadding));
+    CurrentRTLLeftPadding: fallback(value(RTLLeftPadding),value(DefaultRTLLeftPadding));
+    CurrentRTLBottomPadding: fallback(value(RTLBottomPadding),value(DefaultRTLBottomPadding));
+}
+
+.root {
+    -st-states: isMultiday, isSelected, isRightToLeft;
+    position: relative;
+    padding: value(CurrentTopPadding) value(CurrentRightPadding) value(CurrentBottomPadding) value(CurrentLeftPadding);
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.root:isRightToLeft {
+    position: relative;
+    direction: rtl;
+    display: table;
```

Comment:
I'm not sure using `table` is necessary here.
It seems to be a bit of an overkill imo

Reply:
what I said above...

Reply:
So this should be changed to behave similar to this:
https://codepen.io/Jonnybordo/pen/KKVpRGX